### DISCUSSION TITLE
DGBAT Development Update & Community Messaging

### DISCUSSION THREAD

**Discussion by DGBarchie** [DigiByte-Core/digibyte#156] [General]
Hello everyone,

This is Archie from the DigiByte Awareness Team. We have been seeing a great response to content we've been releasing lately, and the community is showing a lot of interest. I want to thank everyone here for their commitment & effort — It's great to see all the work being done! That said, I am not a developer and am unable to answer questions community members have about DigiByte development. 

Across all the different platforms and social media sites, the most common questions center on the general state of development and v8.22 progress. Could you guys please share your thoughts and perspectives? We aim to contextualize the blockchain upgrade process, and share accurate, up-to-date info with the community. 

Many others have also asked: "What can we do to help?"

Would love to know what you guys think, cheers! 🍻

**Comment by saltedlolly** [comment]
Currently work is progressing testing v8 and documenting and fixing bugs. Once these have all been resolved satisfactorily RC3 will be released for hopefully a last round of testing before a final release. How long this will take, I can't say, but the more people we can get to help, the better.

**Reply by DGBarchie** [reply]
Hey Olly, thanks for the response — and happy new year! From updates on pull-requests here on Github & Twitter, I can see quite a bit of work has already been done! Is there something such as a checklist or progress meter of some sort to help visualize what's complete and what still needs attention? if not - I would be glad to aid in conjuring one up.

Also, who exactly can help and how!? What skill sets specifically would come in handy. Many in the community are eager but helping with development is daunting! Would love to get the message out there 👍

Once again, appreciate your input. I will be spending time getting to better understand GitHub and the development process!

**Reply by saltedlolly** [reply]
Firstly I am not a core dev and can't really speak for everyone, but for myself it is really about working until everyone is satisfied that it is the best that we can make it. Only then should we consider it ready for a final release.

Issues, as they are being discovered, are being documented here: https://github.com/DigiByte-Core/digibyte/issues

Fixes to those Issues, in the form of Pull Requests, are being made here: https://github.com/DigiByte-Core/digibyte/pulls

We need help from skilled C++ developers to help make PRs to try and fix the issues that have been identified, review PRs made by other developers, and generally help make v8 as robust as we can make it. Most of us are not C++ developers (myself included), and we are simply doing our best to help within the limits of our abilities. The other thing people can do to simply help test by running a testnet node with the latest test versions of DigiByte Core. Many of the recent Issues that have been identified are thanks to all the volunteers running a testnet node, and collaborating to test and identify problems.

@JongJan88 has been helping to put together compiled binaries with the latest fixes to make testing easier. They can be found here: https://digibytenode.com/testing/ (IMPORTANT: These should be used for testing only!!)

If people want to help test it's a good idea to join the DigiByte Discord and join the chat in the #core_8-22-0_testing channel here: https://discord.com/channels/878200503815782400/1117868419283423293

Also the #𝗗𝗚𝗕𝗖𝗜𝗧 channel here: https://discord.com/channels/878200503815782400/1133815334013509764

This is where most of the discussion is happening around testing.
